misc/azure-release.yml

```diff
@@ -111,15 +111,14 @@ stages:
               export TILEDB_INSTALL=.libtiledb
               export CIBW_ENVIRONMENT="TILEDB_PATH=${TILEDB_INSTALL} TILEDB_WHEEL_BUILD=1"
               # use the requirements_wheel.txt with numpy pins to ensure ABI compatibility
-              export CIBW_BEFORE_TEST="pip install -r misc/requirements_wheel.txt"
               export CIBW_TEST_COMMAND="python -c 'import tiledb'"
               # copy libtiledb into usr/local for audithweel to find
               export CIBW_BEFORE_BUILD="cp -R .libtiledb/* /usr/local"
               ls -lR "${TILEDB_INSTALL}"
 
               python -c "import os; print(os.environ.get('CIBW_ENVIRONMENT', None))"
               git rev-parse HEAD
-              python3 -m pip install --upgrade pip pytest cython pybind11
+              python3 -m pip install --upgrade pip pytest cython
```

Comment:
should add cython to pyproject.toml IMO

Reply:
Sorry made a mistake while rebasing. It actually still requires the `pip install pybind11` here. Thanks for catching.